tests/trips/test_trips.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+
+from api.modules.city.model import City
+from api.modules.trips.model import Trip
+
+
+class TestRemoveUserFromTripAPI(APITestCase):
```

Comment:
By convention we have one test class for one file / class
And each test's name as `test_<method_name>_<condition you're testing>`
Eg: `test_ remove_user_from_trip_success`, `test_ remove_user_from_trip_unauthorized_user`, `test_ remove_user_from_trip_invalid_trip`...

Reply:
@prabhakar267 so shall I name this class as TestTripAPI.
and inside methods as you gave examples.

Reply:
TestTripViews